msgs/src/main/scala/com/thenewmotion/ocpi/msgs/OcpiStatusCode.scala

```diff
@@ -41,6 +41,7 @@ object OcpiStatusCode {
   val PartyAlreadyRegistered = ClientErrorCode(2012) // When POSTing
   val RegistrationNotCompletedYetByParty = ClientErrorCode(2013) // When PUTing or GETing even
   val AuthorizationFailed = ClientErrorCode(2014)
+  val ClientWasNotRegistered = ClientErrorCode(2015)
```

Comment:
Was this in the spec somewhere or you just picked the next number?  I'm not sure where these other error codes came from TBH